lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -424,6 +424,15 @@ void SPIRVEntry::updateModuleVersion() const {
   Module->setMinSPIRVVersion(getRequiredSPIRVVersion());
 }
 
+bool SPIRVEntry::isItAllowedToUseExtension(ExtensionID AskedExtension) const {
```

Comment:
This is more like a helper, but since you can always access to `Module` from any sub-class of `SPIRVEntry` I decided to remove them to avoid possible confusion